shopfloor_reception/services/reception.py

```diff
@@ -158,8 +158,15 @@ def _order_stock_picking(self):
 
     def _scan_document__scan_picking(self, barcode):
         search = self._actions_for("search")
-        picking = search.picking_from_scan(barcode)
+        picking = search.picking_from_scan(barcode, True)
```

Comment:
Since at this stage the only scenario that accepts SO is reception (last to arrive), I set it to false by default in all other cases. Not sure if all other scenarios will have to accept it as well in the future? @jbaudoux